### ISSUE TITLE
version 0.106.1 introduced exporter 401 errors

### ISSUE THREAD

**Issue by ceastman-r7** [open-telemetry/opentelemetry-collector-contrib#34787]
### Component(s)

exporter/prometheusremotewrite

## Description
the prometheus remote write and the otlp http exporter are getting 401 errors when switching helm chart versions from 0.100.0 to 0.101.0 which corresponds to application versions: 0.105.0 to 0.106.1

## Steps to Reproduce
helm install chart version 0.100.0 see successful exports, upgrade to chart version 0.101.0 and see exporters with the same configuration return 401 errors from the endpoint

## Expected Result
exporters are able to successful export data to endpoints

## Actual Result
401 errors returned from endpoints

### Collector version

0.106.1

## Environment
OS: (e.g., "Ubuntu 20.04")
Compiler(if manually compiled): (e.g., "go 14.2")

### OpenTelemetry Collector configuration

```yaml
apiVersion: v1
data:
  relay: |
    exporters:
      datadog:
        api:
          key: ${env:DD_API_KEY}
          site: datadoghq.com
        metrics:
          resource_attributes_as_tags: true
      debug:
        verbosity: detailed
      otlphttp:
        auth:
          authenticator: basicauth/otlp
        endpoint: ${env:otel_otlp_endpoint}
      prometheusremotewrite/external:
        auth:
          authenticator: basicauth/metrics
        endpoint: ${env:otel_metrics_endpoint}
        resource_to_telemetry_conversion:
          enabled: true
        target_info:
          enabled: false
      zipkin:
        default_service_name: unknown-service
        endpoint: http://istio-jaeger-collector.istio-addons.svc.cluster.local:9411/api/v2/spans
        format: proto
    extensions:
      basicauth/metrics:
        client_auth:
          password: ${env:otel_metrics_api}
          username: ${env:otel_metrics_user}
      basicauth/otlp:
        client_auth:
          password: ${env:otel_otlp_api}
          username: ${env:otel_otlp_user}
      health_check:
        endpoint: ${env:MY_POD_IP}:13133
    processors:
      attributes/addtags:
        actions:
        - action: insert
          key: kube_cluster_name
          value: ${env:K8S_CLUSTER_NAME}
        - action: insert
          key: otel_account_alias
          value: kubeplay-staging
      attributes/droptags:
        actions:
        - action: delete
          key: app_kubernetes_io_instance
        - action: delete
          key: app_kubernetes_io_name
        - action: delete
          key: cloud.platform
        - action: delete
          key: cloud.provider
        - action: delete
          key: cloud.region
        - action: delete
          key: cloud_platform
        - action: delete
          key: cloud_provider
        - action: delete
          key: cloud_region
        - action: delete
          key: container.id
        - action: delete
          key: container.image.name
        - action: delete
          key: container.runtime.version
        - action: delete
          key: container_id
        - action: delete
          key: container_image_name
        - action: delete
          key: container_runtime_version
        - action: delete
          key: destination_canonical_revision
        - action: delete
          key: destination_canonical_service
        - action: delete
          key: destination_cluster
        - action: delete
          key: family
        - action: delete
          key: host.id
        - action: delete
          key: host.image.id
        - action: delete
          key: host.name
        - action: delete
          key: host.type
        - action: delete
          key: host_id
        - action: delete
          key: host_image_id
        - action: delete
          key: host_name
        - action: delete
          key: host_type
        - action: delete
          key: http.scheme
        - action: delete
          key: http_scheme
        - action: delete
          key: instance
        - action: delete
          key: internal.ip
        - action: delete
          key: internal_ip
        - action: delete
          key: job
        - action: delete
          key: k8s.cronjob.uid
        - action: delete
          key: k8s.daemonset.uid
        - action: delete
          key: k8s.deployment.uid
        - action: delete
          key: k8s.hpa.uid
        - action: delete
          key: k8s.job.uid
        - action: delete
          key: k8s.namespace.uid
        - action: delete
          key: k8s.node.uid
        - action: delete
          key: k8s.pod.ip
        - action: delete
          key: k8s.pod.uid
        - action: delete
          key: k8s.replicaset.uid
        - action: delete
          key: k8s.statefulset.uid
        - action: delete
          key: k8s_cronjob_uid
        - action: delete
          key: k8s_daemonset_uid
        - action: delete
          key: k8s_deployment_uid
        - action: delete
          key: k8s_hpa_uid
        - action: delete
          key: k8s_job_uid
        - action: delete
          key: k8s_namespace_uid
        - action: delete
          key: k8s_node_uid
        - action: delete
          key: k8s_pod_ip
        - action: delete
          key: k8s_pod_uid
        - action: delete
          key: k8s_replicaset_uid
        - action: delete
          key: k8s_statefulset_uid
        - action: delete
          key: kubelet.version
        - action: delete
          key: kubelet_version
        - action: delete
          key: net.host.name
        - action: delete
          key: net.host.port
        - action: delete
          key: net_host_name
        - action: delete
          key: net_host_port
        - action: delete
          key: os.image
        - action: delete
          key: os.type
        - action: delete
          key: os_image
        - action: delete
          key: os_type
        - action: delete
          key: otel.scope.name
        - action: delete
          key: otel_scope_name
        - action: delete
          key: plugin
        - action: delete
          key: pod_template_hash
        - action: delete
          key: policy.background.mode
        - action: delete
          key: policy.type
        - action: delete
          key: policy_background_mode
        - action: delete
          key: policy_type
        - action: delete
          key: proto
        - action: delete
          key: provider.id
        - action: delete
          key: provider_id
        - action: delete
          key: request_protocol
        - action: delete
          key: resource.request.operation
        - action: delete
          key: resource_request_operation
        - action: delete
          key: response_flags
        - action: delete
          key: rule.execution.cause
        - action: delete
          key: rule.type
        - action: delete
          key: rule_execution_cause
        - action: delete
          key: rule_type
        - action: delete
          key: server
        - action: delete
          key: service.instance
        - action: delete
          key: service.instance.id
        - action: delete
          key: service.name
        - action: delete
          key: service_instance
        - action: delete
          key: service_instance_id
        - action: delete
          key: service_name
        - action: delete
          key: source_canonical_revision
        - action: delete
          key: source_canonical_service
        - action: delete
          key: source_cluster
        - action: delete
          key: source_principal
        - action: delete
          key: system.uuid
        - action: delete
          key: system_uuid
        - action: delete
          key: uid
        - action: delete
          key: zone
        - action: delete
          key: zones
      batch:
        send_batch_max_size: 1000
        send_batch_size: 100
        timeout: 10s
      filter/dropdatapoints:
        error_mode: ignore
        metrics:
          datapoint:
          - metric.name == "istio_tcp_connections_opened_total" and attributes["destination_service_name"]
            != "BlackHoleCluster"
          - metric.name == "istio_tcp_connections_opened_total" and attributes["source_workload"]
            == "prometheus-grafana"
          - metric.name == "k8s.replicaset.desired" and value_int == 0
          - IsMatch(metric.name,  "istio_.+") and not IsMatch(attributes["namespace"],
            "---not-a-valid-namespace---" )
          - IsMatch(metric.name,  "istio_.+") and IsMatch("---not-a-valid-namespace---",
            "---not-a-valid-namespace---")
      filter/dropprommetricsregex:
        metrics:
          exclude:
            match_type: regexp
            metric_names:
            - scrape_.*
            - up
      filter/includemetrics:
        metrics:
          include:
            match_type: strict
            metric_names:
            - coredns_cache_hits_total
            - coredns_dns_request_duration_seconds
            - coredns_dns_requests_total
            - coredns_dns_responses_total
            - istio_tcp_connections_opened_total
            - k8s.container.cpu_request_utilization
            - k8s.container.cpu_request
            - k8s.container.memory_limit_utilization
            - k8s.container.memory_request_utilization
            - k8s.container.memory_request
            - k8s.container.ready
            - k8s.container.restarts
            - k8s.daemonset.desired_scheduled_nodes
            - k8s.daemonset.ready_nodes
            - k8s.deployment.available
            - k8s.deployment.desired
            - k8s.node.cpu.usage
            - k8s.node.memory.usage
            - k8s.node.network.errors
            - k8s.node.network.io
            - k8s.pod.cpu.usage
            - k8s.pod.memory.usage
            - k8s.pod.network.errors
            - k8s.pod.network.io
            - k8s.pod.phase
            - k8s.replicaset.available
            - k8s.replicaset.desired
            - k8s.statefulset.desired_pods
            - k8s.statefulset.ready_pods
            - kube_node_info
            - kube_node_spec_unschedulable
            - kube_node_status_allocatable
            - kube_node_status_capacity
            - kube_node_status_condition
            - kube_pod_container_info
            - kube_pod_container_resource_limits
            - kube_pod_container_resource_requests
            - kube_pod_container_status_ready
            - kube_pod_container_status_running
            - kube_pod_container_status_terminated
            - kube_pod_container_status_waiting_reason
            - kube_pod_container_status_waiting
            - kube_pod_info
            - kube_pod_init_container_info
            - kube_pod_init_container_status_ready
            - kube_pod_init_container_status_running
            - kube_pod_init_container_status_terminated
            - kube_pod_init_container_status_waiting
            - kube_service_info
            - policy_report_result
      filter/istioingresshealtcheck:
        error_mode: ignore
        traces:
          span:
          - attributes["user_agent"] == "ELB-HealthChecker/2.0"
      k8sattributes:
        auth_type: serviceAccount
        extract:
          labels:
          - from: pod
            key: app.kubernetes.io/name
            tag_name: kube_app_name
          - from: pod
            key: app.kubernetes.io/instance
            tag_name: kube_app_instance
          - from: pod
            key: app.kubernetes.io/version
            tag_name: kube_app_version
          - from: pod
            key: app.kubernetes.io/component
            tag_name: kube_app_component
          metadata:
          - container.image.tag
          - k8s.container.name
          - k8s.pod.name
          - k8s.pod.uid
          - k8s.deployment.name
          - k8s.node.name
          - k8s.namespace.name
          - k8s.daemonset.name
          - k8s.job.name
          - k8s.cronjob.name
          - k8s.statefulset.name
        passthrough: false
        pod_association:
        - sources:
          - from: resource_attribute
            name: k8s.pod.uid
      memory_limiter:
        check_interval: 5s
        limit_percentage: 80
        spike_limit_percentage: 25
      metricstransform/droppassingresults:
        transforms:
        - action: update
          include: policy_report_result
          operations:
          - action: delete_label_value
            label: status
            label_value: pass
      resource/droptags:
        attributes:
        - action: delete
          key: app_kubernetes_io_instance
        - action: delete
          key: app_kubernetes_io_name
        - action: delete
          key: cloud.platform
        - action: delete
          key: cloud.provider
        - action: delete
          key: cloud.region
        - action: delete
          key: cloud_platform
        - action: delete
          key: cloud_provider
        - action: delete
          key: cloud_region
        - action: delete
          key: container.id
        - action: delete
          key: container.image.name
        - action: delete
          key: container.runtime.version
        - action: delete
          key: container_id
        - action: delete
          key: container_image_name
        - action: delete
          key: container_runtime_version
        - action: delete
          key: destination_canonical_revision
        - action: delete
          key: destination_canonical_service
        - action: delete
          key: destination_cluster
        - action: delete
          key: family
        - action: delete
          key: host.id
        - action: delete
          key: host.image.id
        - action: delete
          key: host.name
        - action: delete
          key: host.type
        - action: delete
          key: host_id
        - action: delete
          key: host_image_id
        - action: delete
          key: host_name
        - action: delete
          key: host_type
        - action: delete
          key: http.scheme
        - action: delete
          key: http_scheme
        - action: delete
          key: instance
        - action: delete
          key: internal.ip
        - action: delete
          key: internal_ip
        - action: delete
          key: job
        - action: delete
          key: k8s.cronjob.uid
        - action: delete
          key: k8s.daemonset.uid
        - action: delete
          key: k8s.deployment.uid
        - action: delete
          key: k8s.hpa.uid
        - action: delete
          key: k8s.job.uid
        - action: delete
          key: k8s.namespace.uid
        - action: delete
          key: k8s.node.uid
        - action: delete
          key: k8s.pod.ip
        - action: delete
          key: k8s.pod.uid
        - action: delete
          key: k8s.replicaset.uid
        - action: delete
          key: k8s.statefulset.uid
        - action: delete
          key: k8s_cronjob_uid
        - action: delete
          key: k8s_daemonset_uid
        - action: delete
          key: k8s_deployment_uid
        - action: delete
          key: k8s_hpa_uid
        - action: delete
          key: k8s_job_uid
        - action: delete
          key: k8s_namespace_uid
        - action: delete
          key: k8s_node_uid
        - action: delete
          key: k8s_pod_ip
        - action: delete
          key: k8s_pod_uid
        - action: delete
          key: k8s_replicaset_uid
        - action: delete
          key: k8s_statefulset_uid
        - action: delete
          key: kubelet.version
        - action: delete
          key: kubelet_version
        - action: delete
          key: net.host.name
        - action: delete
          key: net.host.port
        - action: delete
          key: net_host_name
        - action: delete
          key: net_host_port
        - action: delete
          key: os.image
        - action: delete
          key: os.type
        - action: delete
          key: os_image
        - action: delete
          key: os_type
        - action: delete
          key: otel.scope.name
        - action: delete
          key: otel_scope_name
        - action: delete
          key: plugin
        - action: delete
          key: pod_template_hash
        - action: delete
          key: policy.background.mode
        - action: delete
          key: policy.type
        - action: delete
          key: policy_background_mode
        - action: delete
          key: policy_type
        - action: delete
          key: proto
        - action: delete
          key: provider.id
        - action: delete
          key: provider_id
        - action: delete
          key: request_protocol
        - action: delete
          key: resource.request.operation
        - action: delete
          key: resource_request_operation
        - action: delete
          key: response_flags
        - action: delete
          key: rule.execution.cause
        - action: delete
          key: rule.type
        - action: delete
          key: rule_execution_cause
        - action: delete
          key: rule_type
        - action: delete
          key: server
        - action: delete
          key: service.instance
        - action: delete
          key: service.instance.id
        - action: delete
          key: service.name
        - action: delete
          key: service_instance
        - action: delete
          key: service_instance_id
        - action: delete
          key: service_name
        - action: delete
          key: source_canonical_revision
        - action: delete
          key: source_canonical_service
        - action: delete
          key: source_cluster
        - action: delete
          key: source_principal
        - action: delete
          key: system.uuid
        - action: delete
          key: system_uuid
        - action: delete
          key: uid
        - action: delete
          key: zone
        - action: delete
          key: zones
      resourcedetection:
        detectors:
        - env
        - eks
        - ec2
        - system
        override: false
        timeout: 2s
      transform:
        error_mode: ignore
        metric_statements:
        - context: datapoint
          statements:
          - set(attributes["pod_phase"], "pending") where metric.name == "k8s.pod.phase"
            and value_int == 1
          - set(attributes["pod_phase"], "running") where metric.name == "k8s.pod.phase"
            and value_int == 2
          - set(attributes["pod_phase"], "succeeded") where metric.name == "k8s.pod.phase"
            and value_int == 3
          - set(attributes["pod_phase"], "failed") where metric.name == "k8s.pod.phase"
            and value_int == 4
          - set(attributes["pod_phase"], "unknown") where metric.name == "k8s.pod.phase"
            and value_int == 5
    receivers:
      jaeger:
        protocols:
          grpc:
            endpoint: ${env:MY_POD_IP}:14250
          thrift_compact:
            endpoint: ${env:MY_POD_IP}:6831
          thrift_http:
            endpoint: ${env:MY_POD_IP}:14268
      k8s_cluster:
        allocatable_types_to_report:
        - cpu
        - memory
        - storage
        - ephemeral-storage
        collection_interval: 30s
        metrics:
          k8s.node.condition:
            enabled: true
          k8s.pod.status_reason:
            enabled: true
        node_conditions_to_report:
        - Ready
        - MemoryPressure
        - DiskPressure
        - NetworkUnavailable
      otlp:
        protocols:
          grpc:
            endpoint: ${env:MY_POD_IP}:4317
          http:
            endpoint: ${env:MY_POD_IP}:4318
      prometheus:
        config:
          scrape_configs:
          - job_name: otel-coredns
            scrape_interval: 60s
            static_configs:
            - targets:
              - kube-dns.kube-system:9153
          - job_name: otel-kube-state-metrics
            scrape_interval: 60s
            static_configs:
            - targets:
              - otel-kube-state-metrics:8080
          - job_name: kyverno
            scrape_interval: 60s
            static_configs:
            - targets:
              - policy-reporter.kyverno.svc.cluster.local:8080
          - honor_labels: true
            job_name: kube-proxy
            kubernetes_sd_configs:
            - role: pod
            relabel_configs:
            - action: keep
              regex: kube-system/kube-proxy.+
              separator: /
              source_labels:
              - __meta_kubernetes_namespace
              - __meta_kubernetes_pod_name
            - action: replace
              regex: (.+?)(\\:\\d+)?
              replacement: $$1:10249
              source_labels:
              - __address__
              target_label: __address__
            scrape_interval: 60s
          - honor_labels: true
            job_name: kubernetes-pods-egress
            kubernetes_sd_configs:
            - role: pod
            metric_relabel_configs:
            - action: keep
              regex: istio_tcp_connections_opened_total
              source_labels:
              - __name__
            relabel_configs:
            - action: keep
              regex: true
              source_labels:
              - __meta_kubernetes_pod_annotation_prometheus_io_scrape
            - action: drop
              regex: true
              source_labels:
              - __meta_kubernetes_pod_annotation_prometheus_io_scrape_slow
            - action: replace
              regex: (https?)
              source_labels:
              - __meta_kubernetes_pod_annotation_prometheus_io_scheme
              target_label: __scheme__
            - action: replace
              regex: (.+)
              source_labels:
              - __meta_kubernetes_pod_annotation_prometheus_io_path
              target_label: __metrics_path__
            - action: replace
              regex: (.+?)(?::\d+)?;(\d+)
              replacement: $$1:$$2
              source_labels:
              - __address__
              - __meta_kubernetes_pod_annotation_prometheus_io_port
              target_label: __address__
            - action: labelmap
              regex: __meta_kubernetes_pod_annotation_prometheus_io_param_(.+)
              replacement: __param_$$1
            - action: labelmap
              regex: __meta_kubernetes_pod_label_(.+)
            - action: drop
              regex: Pending|Succeeded|Failed|Completed
              source_labels:
              - __meta_kubernetes_pod_phase
            scrape_interval: 60s
          - honor_labels: true
            job_name: istio-metrics
            kubernetes_sd_configs:
            - role: pod
            metric_relabel_configs:
            - action: drop
              regex: cert_manager_.*|citadel_.*|envoy_.*|kiali_.*|kube_.*|pilot_.*|scrape_.*|sidecar_.*|up
              source_labels:
              - __name__
            - action: labeldrop
              regex: chart|destination_app|destination_version|heritage|.*operator.*|istio.*|release|security_istio_io_.*|service_istio_io_.*|sidecar_istio_io_inject|source_app|source_version
            relabel_configs:
            - action: keep
              regex: true
              source_labels:
              - __meta_kubernetes_pod_annotation_prometheus_io_scrape
            - action: drop
              regex: true
              source_labels:
              - __meta_kubernetes_pod_annotation_prometheus_io_scrape_slow
            - action: replace
              regex: (https?)
              source_labels:
              - __meta_kubernetes_pod_annotation_prometheus_io_scheme
              target_label: __scheme__
            - action: replace
              regex: (.+)
              source_labels:
              - __meta_kubernetes_pod_annotation_prometheus_io_path
              target_label: __metrics_path__
            - action: replace
              regex: (.+?)(?::\d+)?;(\d+)
              replacement: $$1:$$2
              source_labels:
              - __address__
              - __meta_kubernetes_pod_annotation_prometheus_io_port
              target_label: __address__
            - action: labelmap
              regex: __meta_kubernetes_pod_annotation_prometheus_io_param_(.+)
              replacement: __param_$$1
            - action: labelmap
              regex: __meta_kubernetes_pod_label_(.+)
            - action: replace
              source_labels:
              - __meta_kubernetes_namespace
              target_label: namespace
            - action: drop
              regex: Pending|Succeeded|Failed|Completed
              source_labels:
              - __meta_kubernetes_pod_phase
            scrape_interval: 30s
      zipkin:
        endpoint: ${env:MY_POD_IP}:9411
    service:
      extensions:
      - basicauth/metrics
      - basicauth/otlp
      - health_check
      pipelines:
        logs:
          exporters:
          - debug
          processors:
          - memory_limiter
          - batch
          receivers:
          - otlp
        metrics:
          exporters:
          - datadog
          processors:
          - attributes/addtags
          - resourcedetection
          - k8sattributes
          - filter/includemetrics
          - filter/dropprommetricsregex
          - filter/dropdatapoints
          - metricstransform/droppassingresults
          - resource/droptags
          - attributes/droptags
          - transform
          - batch
          receivers:
          - k8s_cluster
          - prometheus
        metrics/clustermetricsexternal:
          exporters:
          - prometheusremotewrite/external
          processors:
          - attributes/addtags
          - resourcedetection
          - k8sattributes
          - filter/includemetrics
          - filter/dropprommetricsregex
          - filter/dropdatapoints
          - metricstransform/droppassingresults
          - resource/droptags
          - attributes/droptags
          - batch
          receivers:
          - k8s_cluster
          - prometheus
        traces:
          exporters:
          - zipkin
          processors:
          - filter/istioingresshealtcheck
          receivers:
          - otlp
          - zipkin
        traces/external:
          exporters:
          - otlphttp
          processors:
          - filter/istioingresshealtcheck
          receivers:
          - otlp
          - zipkin
      telemetry:
        logs:
          encoding: json
        metrics:
          address: ${env:MY_POD_IP}:8888
          level: detailed
kind: ConfigMap
metadata:
  annotations:
    meta.helm.sh/release-name: otel-deployment
    meta.helm.sh/release-namespace: platform
  labels:
    app.kubernetes.io/instance: otel-deployment
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: opentelemetry-collector
    app.kubernetes.io/version: 0.105.0
    helm.sh/chart: opentelemetry-collector-0.100.0
  name: otel-deploy-agent
  namespace: platform
```

### Log output

```shell
otel-ds-agent-xpp4m opentelemetry-collector {"level":"error","ts":1724247145.6110976,"caller":"exporterhelper/queue_sender.go:92","msg":"Exporting failed. Dropping data.","kind":"exporter","data_type":"logs","name":"otlphttp","error":"not retryable error: Permanent error: rpc error: code = Unauthenticated desc = error exporting items, request to https://otlp-gateway-prod-us-west-0.grafana.net/otlp/v1/logs responded with HTTP Status Code 401","dropped_items":1,"stacktrace":"go.opentelemetry.io/collector/exporter/exporterhelper.newQueueSender.func1\n\tgo.opentelemetry.io/collector/exporter@v0.106.1/exporterhelper/queue_sender.go:92\ngo.opentelemetry.io/collector/exporter/internal/queue.(*boundedMemoryQueue[...]).Consume\n\tgo.opentelemetry.io/collector/exporter@v0.106.1/internal/queue/bounded_memory_queue.go:52\ngo.opentelemetry.io/collector/exporter/internal/queue.(*Consumers[...]).Start.func1\n\tgo.opentelemetry.io/collector/exporter@v0.106.1/internal/queue/consumers.go:43"}
```

**Comment by iblancasa** [comment]
Could it be the problem in the `basicauth` extension instead of in the `prometheusremotewrite` exporter?
Even, can you verify the environment variables you are using are properly set? Maybe there is something that changed in the Helm chart that is not setting properly the environment variables.

**Comment by rapphil** [comment]
The error log that you provided point sto a OTLP endpoint and not to a prometheus remote write endpoint:

```
request to https://otlp-gateway-prod-us-west-0.grafana.net/otlp/v1/logs responded with HTTP Status Code 401
```

**Comment by ceastman-r7** [comment]
it happens to both the prometheus remote write exporter and the otlphttp exporter. they both receiver 401 errors back from the endpoints when using 0.106.1 version of the application.

**Comment by iblancasa** [comment]
> it happens to both the prometheus remote write exporter and the otlphttp exporter. they both receiver 401 errors back from the endpoints when using 0.106.1 version of the application.

Can you verify the environment variables you are using are properly set? Maybe there is something that changed in the Helm chart that is not setting properly the environment variables.

**Comment by rapphil** [comment]
IT is also interesting that in the configuration that you shared, there are no exporters besides the debug exporter, however from the log that you shared, it seems that it is trying to access a url for the logs signal type...

**Comment by ceastman-r7** [comment]
at the top of the config:
`
    exporters:
      datadog:
        api:
          key: ${env:DD_API_KEY}
          site: datadoghq.com
        metrics:
          resource_attributes_as_tags: true
      debug:
        verbosity: detailed
      otlphttp:
        auth:
          authenticator: basicauth/otlp
        endpoint: ${env:otel_otlp_endpoint}
      prometheusremotewrite/external:
        auth:
          authenticator: basicauth/metrics
        endpoint: ${env:otel_metrics_endpoint}
        resource_to_telemetry_conversion:
          enabled: true
        target_info:
          enabled: false
      zipkin:
        default_service_name: unknown-service
        endpoint: http://istio-jaeger-collector.istio-addons.svc.cluster.local:9411/api/v2/spans
        format: proto
`

**Comment by ceastman-r7** [comment]
It doesn't appear to be related to the helm chart. I have helm chart version 0.100.0 installed which installs the app version of 0.105.0. When I check the application logs there are no 401 errors from the exporters.

I then changed the deployment to use the image tag version of 0.106.1 and now I have 401 errors from the exporters, and I see 401 errors from the otlphttp and the prometheus remote write exporters.

**Comment by rapphil** [comment]
>IT is also interesting that in the configuration that you shared, there are no exporters besides the debug exporter, however from the log that you shared, it seems that it is trying to access a url for the logs signal type...

I meant here:

```
      pipelines:
        logs:
          exporters:
          - debug
          processors:
          - memory_limiter
          - batch
          receivers:
          - otlp
````

**Comment by ceastman-r7** [comment]
ah, i have a separate config for the daemonset where the log collection happens.

**Comment by ceastman-r7** [comment]
@rapphil Anything I can do to help troubleshoot this?

**Comment by ceastman-r7** [comment]
@dashpole Is there anyone else available to look at this?

**Comment by dashpole** [comment]
Can you provide the error from the prometheusremotewrite exporter?

**Comment by ceastman-r7** [comment]
I just installed the latest helm chart / app version - 0.105.1 / 0.109.0 and still see the same issue.

otel-deploy-agent-7cf97d46b7-qjz98 opentelemetry-collector {"level":"error","ts":1726670333.6662629,"caller":"exporterhelper/queue_sender.go:92","msg":"Exporting failed. Dropping data.","kind":"exporter","data_type":"metrics","name":"prometheusremotewrite/external","error":"Permanent error: Permanent error: Permanent error: remote write returned HTTP status 401 Unauthorized; err = %!w(<nil>): {\"status\":\"error\",\"error\":\"authentication error: invalid authentication credentials\"}","dropped_items":210,"stacktrace":"go.opentelemetry.io/collector/exporter/exporterhelper.newQueueSender.func1\n\tgo.opentelemetry.io/collector/exporter@v0.109.0/exporterhelper/queue_sender.go:92\ngo.opentelemetry.io/collector/exporter/internal/queue.(*boundedMemoryQueue[...]).Consume\n\tgo.opentelemetry.io/collector/exporter@v0.109.0/internal/queue/bounded_memory_queue.go:52\ngo.opentelemetry.io/collector/exporter/internal/queue.(*Consumers[...]).Start.func1\n\tgo.opentelemetry.io/collector/exporter@v0.109.0/internal/queue/consumers.go:43"}


the same exact configuration works fine with chart 0.100.0 / app 0.105.0

could it be related to this? https://github.com/open-telemetry/opentelemetry-collector/issues/10750

**Comment by ceastman-r7** [comment]
here are some config from the configmap and the deployment yaml:

`
from otel config map

    exporters:
      prometheusremotewrite/external:
        auth:
          authenticator: basicauth/metrics
        endpoint: ${env:otel_metrics_endpoint}
        resource_to_telemetry_conversion:
          enabled: true
        target_info:
          enabled: false
    extensions:
      basicauth/metrics:
        client_auth:
          password: ${env:otel_metrics_api}
          username: ${env:otel_metrics_user}




from pod deployment
        - name: otel_metrics_api
          valueFrom:
            secretKeyRef:
              key: api
              name: otel-external-metrics-auth
        - name: otel_metrics_endpoint
          valueFrom:
            secretKeyRef:
              key: endpoint
              name: otel-external-metrics-auth
        - name: otel_metrics_user
          valueFrom:
            secretKeyRef:
              key: user
              name: otel-external-metrics-auth      
`

**Comment by dashpole** [comment]
The fix to https://github.com/open-telemetry/opentelemetry-collector/issues/10750 was released with v0.107.0, so if upgrading to that didn't work for you, then it probably wasn't that.

**Comment by ceastman-r7** [comment]
I just tried going to helm chart version 0.108.0 and I still see 401 errors in the logs with regards to the otlphttp and the prometheusremotewrite exporters still.
